### DISCUSSION TITLE
Empty name for full page renders

### DISCUSSION THREAD

**Discussion by damms005** [livewire/livewire#3034] [7. Bugs (archived)]
### Description
I `livewire:make` a full page component. When I call a method from the component, the ensuing xhr request has the `name` property is blank, like in the screenshot below, causing an error 404 pop from livewire.
![image](https://user-images.githubusercontent.com/9839355/115994792-5d2ddf00-a5d0-11eb-934c-54bee080d57d.png)

### Exact steps to reproduce
1. `livewire:make` full page component.
2. Register a route for the component in routes file
3. Create a button in the component's view file that make calls an action on the component
4. Error 404 pops up (because of empty string value posted for `name` as in the screenshot above)

### Stripped-down, copy-pastable code snippets
```
  <div class="flex-grow p-5 m-10 text-white bg-red-500 rounded">
	<h1>Tim Posts</h1>
	<div>
		@foreach ($timPosts as $timPost)
		<div class="p-2 my-5 bg-red-400 rounded ">
			<span class="text-gray-400">{{$loop->index + 1}}</span>
			{{$timPost->post_title}}
			<button wire:key="$timPost->title" wire:click="convertTimPostToFortunatoPost({{$timPost->id}})" class="p-2 px-3 bg-red-200 rounded hover:bg-red-800">convert</button>
		</div>
		@endforeach
	</div>
</div>
```

```
public function convertTimPostToFortunatoPost($timModelId)
{
	/**
	* @var TimPost
	*/
	$timModel = TimPost::find($timModelId);

	throw_if(is_null($timModel), new \Exception("Could not find TimPost with id={$timModelId}"));

	$fortunato = $timModel->convertToFortunato();

	$fortunato->save();
}
```

**Context**
- Livewire version: 2.3.5
- Laravel version: 8.18.1
- Alpine version: nil
- Browser: Chrome

**Comment by ph7jack** [comment]
Update to latest livewire version
